### PR TITLE
Fix prop type errors on MAP

### LINKED ISSUE #4396: Prop warnings on candy xml transformed paragraph blocks
**Describe the bug**
Prop warnings appear when rendering paragraphs that were candy xml

image
image
image

**To Reproduce**

1. Go to http://localhost.bbc.com:7080/igbo/afirika-23252735
2. See error

**Expected behavior**
No prop warnings

**Screenshots**

**Testing notes**

- ~[ ] This bug fix is expected to need manual testing.~

**Additional context**

### LINKED ISSUE #4397: Prop warning on CPS asset links
**Describe the bug**
A prop warning appear when rendering links from candy xml

image

**To Reproduce**

1. Go to http://localhost.bbc.com:7080/igbo/afirika-23252735
2. See error

**Expected behavior**
No prop warnings

**Screenshots**

**Testing notes**

- ~[ ] This bug fix is expected to need manual testing.~

**Additional context**

### LINKED ISSUE #4405: PropType errors on MAP pages
**Describe the bug**
There are propType warnings about the data being passed to the Text, Paragraph and InlineLink containers. 

**To Reproduce**

1. `npm run dev` 
2. Visit http://localhost.bbc.com:7080/pidgin/23248703
3. Open console and see errors

**Expected behavior**

**Screenshots**
image

**Additional context**
There should be no propType errors on MAPs, and onward journeys to MAPs

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #4405
Resolves #4397
Resolves #4396

**Overall change:** Pulls in `rich-text-transforms@1.1.0` which fixed the propType warnings as it now sets the `isExternal` field on the `urlLink` block which is a required propType. 

**Code changes:**
- Bumps a package
- Updates snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- ~[ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)~
- ~[ ] This PR requires manual testing~

### CODE REVIEW

File: data/pidgin/cpsAssets/23248703.json [hunk 1]
```diff
@@ -209,31 +209,31 @@
               "type": "listItem"
             },
             {
-              "text": "<link><caption>this is the link text</caption><altText>this is the alt text </altText><url href=\"https://bbc.com/pidgin\" platform=\"highweb\"/><url href=\"https://bbc.com/pidgin\" platform=\"enhancedmobile\"/></link>",
+              "text": "<link><caption>this is the link text</caption><altText>this is the alt text </altText><url href=\"https://www.bbc.com/pidgin\" platform=\"highweb\"/><url href=\"https://www.bbc.com/pidgin\" platform=\"enhancedmobile\"/></link>",
```
Comment (pjlee11) [inline data/pidgin/cpsAssets/23248703.json:212]: Just adding `www.` prefix before `bbc.com`

File: src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/index.test.js [hunk 2]
```diff
@@ -52,6 +58,38 @@ describe('convertToOptimoBlocks', () => {
                 text: 'A plain text paragraph',
               },
             ]),
+            {
+              type: 'text',
```
Comment (pjlee11) [inline src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/index.test.js:62]: I'm going to pull this out into nicer fixtures in a follow up PR.

File: data/pidgin/cpsAssets/23248703.json (hunk 1 above)
Comment (dr3) [inline data/pidgin/cpsAssets/23248703.json:212]: Can you write a psammead issue to support `https://bbc.com` as internal please? Likely need to check `https://` or `http://` or `.` before `'bbc'`

**Review by dr3 — APPROVED** [review]

File: data/pidgin/cpsAssets/23248703.json (hunk 1 above)
Comment (pjlee11) [reply data/pidgin/cpsAssets/23248703.json:212]: Done: https://github.com/bbc/psammead/issues/2496

**Review by pharingee — APPROVED** [review]
LGTM

**Pull request merged**